packages/web-monetization-polyfill-utils/src/lib/MonetizationTagObserver.ts

```diff
@@ -3,7 +3,7 @@ import { v4 as uuid } from 'uuid'
 import { whenDocumentReady } from './whenDocumentReady'
 
 // eslint-disable-next-line @typescript-eslint/no-empty-function,@typescript-eslint/no-unused-vars
-const debug = (...args: unknown[]) => {}
+const debug = (..._: unknown[]) => {}
```

Comment:
How does this work? Really just curious.. I don't think I've seen this pattern before.

Reply:
using `_` for unused variables is somewhat of a convention 
are you talking about the `...` syntax ? [rest params](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters)

essentially the `debug` function is a noop and you can change it to console.log while debugging
very "poor man's x" but it does the trick!